statsmodels/stats/tests/test_lilliefors.py

```diff
@@ -0,0 +1,26 @@
+from numpy.testing import assert_, assert_raises
+from scipy import stats
+import nose
+from statsmodels.stats._lilliefors import lilliefors
+import numpy as np
+
+class TestLilliefors(object):
```

Comment:
our unit tests usually verify the results against other packages mainly R or Stata if an equivalent function is available.
Those unit test should catch things like the wrong scale above.